web/src/angular/src/app/shared/user-layout/user-layout.component.html

```diff
@@ -0,0 +1,142 @@
+<div class="content">
+    <div>
+        <div
+            class="relative z-40 transition-[visibility] duration-[0]"
+            role="dialog"
+            aria-modal="true"
+            [ngClass]="
+                isCollapsed ? 'invisible delay-300' : 'visible md:hidden'
+            "
```

Comment:
Nit: single line string?

Reply:
Fixed from prettier like that. Probably because of line length.